src/style/properties.js

```diff
@@ -349,6 +349,7 @@ const styfn = {};
     { name: 'taxi-turn', type: t.bidirectionalSizeMaybePercent, triggersBounds: diff.any },
     { name: 'taxi-turn-min-distance', type: t.size, triggersBounds: diff.any },
     { name: 'taxi-direction', type: t.axisDirection, triggersBounds: diff.any },
+    { name: 'taxi-radius', type: t.number, triggersBounds: diff.any },
```

Comment:
Great name.  It should be documented: https://github.com/cytoscape/cytoscape.js/blob/unstable/documentation/md/style.md